doc/version-history.rst

```diff
@@ -18,6 +19,10 @@ v19.0.0
 
 * Add MTOODS CSC
 
+  * MTM2:
```

Comment:
I think there is an extra indentation here.

Reply:
Fixed in the commit: [025e815](https://github.com/lsst-ts/ts_xml/pull/731/commits/025e8155ca7bc350d930f958be351c7888cd6e83). Thanks!